assets/i18n/es.json

```diff
@@ -1743,6 +1743,7 @@
 	"I18N_TOPNAV_ADMIN_PAGE": "Página de administración",
 	"I18N_TOPNAV_ANDROID_APP_DESCRIPTION": "La primera aplicación de Oppia para Android ya está disponible en <strong>Inglés</strong> y <strong>Portugués brasileño</strong>. ¡Pruébala y envíanos tus comentarios!",
 	"I18N_TOPNAV_ANDROID_APP_HEADING": "Aplicación Android",
+	"I18N_TOPNAV_BLOG": "Blog",
```

Comment:
We only edit `en.json` and `qqq.json` files. The other files are updated through translatewiki.
https://github.com/oppia/oppia/wiki/Adding-new-translations-for-i18n